### DISCUSSION TITLE
中国农业大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#24] [University]
> 回答来自群友，我仅负责搬运

分东（新）、西（旧）两个校区，新生均在东区上大一，部分学院大二转西区

**宿舍是上床下桌吗**
男生宿舍均上床下桌（6人间）；西区女生宿舍均是（部分6人间，部分4人间），东区有部分屋不是

**教室和宿舍有没有空调**
宿舍均有空调，东西区部分教学楼无空调（有大吊扇，不过人多了没啥用）

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
（说到浴室，那就不得不提给了大家一个坦诚相见的机会的澡堂）东区仅女生宿舍有独卫（楼分A/B座，仅A座独卫带浴室，热水供应时间18点\~23点），男生楼一层有澡堂，校内另有一大澡堂（19年翻修完毕，有挡板；距离男生楼300m左右、距离女生楼100m左右）；西区男生楼无独卫，女生楼不详，唯一的澡堂（男女澡堂就隔一个路口）距男生楼3\~5min走路时间，距女生楼4\~6min走路时间；男生楼卫生间都与水房连接，水房开水机24h供应开水；

**有早自习、晚自习吗**
自习是自己的事情，学校不管

**有晨跑吗**
没晨跑

**每学期跑步打卡的要求是多少公里，可以骑车吗**
用运动世界校园（好像还能KEEP），限制里程男生每天2km，女生1.5km；男生60及格80满分，女生40及格60满分；可以骑车，打开完了揣裤兜里就可以撒欢了（说是有人抓不过是纯扯淡，顶多有老师抓那种在操场跑道上骑车的牛人）

**寒暑假放多久，每年小学期有多长**
算是北京放假比较晚开学比较早的学校了，小学期看各个学院如何，生物学院的小学期可以干掉半个月

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许，东区不知道还能不能送到宿舍楼下；西区统一送西北门（也只有这个门能一般进出），距男生宿舍2\~3min骑车时间，5\~6min走路时间，距女生宿舍3\~4min骑车时间、6\~7min走路时间

**学校附近有地铁站吗**
16号线农大南路站距西北门4min走路时间

**宿舍楼有洗衣机吗**
有公共洗衣机，有桶自洁功能（洗前先通一遍臭氧消毒），宿舍阳台有晾衣服用的大横杆子，需要自购晾衣架和长杆

**校园网怎么样**
目前20GB免费，之后1￥/1GB，东区网哪儿都好，西区男生楼（女生楼不详）里进楼就废，宿舍里桌下有网口（有的坏了）需要自购网线、路由器；有eduroam

**每天断电断网吗，几点开始断**
电/网目前均不断供

**食堂价格贵吗，会吃出异物吗**
农大特喜欢吹自己的食堂，不过确实也挺好的——味道以及价格（平价，十块左右一顿可以），个人没吃出过异物，在朋友圈里两年以来也就看到过不超过五次有人吃出过异物；别听他们瞎扯，少去公寓第二食堂，消费高好吃的其实也不多（有意报考可以 @NozomiPower (Telegram)，我可以安利点好吃的）

**洗澡热水供应时间**
浴室开放时间15:30\~23:00

**校园内可以骑电瓶车吗，电池在哪能充电**
可以骑，校内有充电桩（西区校内有大量青桔单车，较大量哈罗，少量摩拜，东区大概也是如此）

**Comment by Zeitlupe202** [comment]
东区那边不能送到宿舍楼下啦。但是东南门和东门两个收外卖的地方离宿舍楼挺近的，下楼走走就到了。校园网准确地来说是1元1000M（怨念）。
